src/utils/materialHelper.js

```diff
@@ -4,6 +4,9 @@ import * as THREE from 'three';
 // mat-phong
 // "color: red; emissive: blue; specular: yellow; opacity: 0.5; shininess: 100; wireframe: true"
 
+// TODO - this should not be in this folder
+// TODO - is this even still needed? i dont see it used in the rest of the code
```

Comment:
@michaelthatsit - (the second TODO line - 8) quick question for you

Reply:
lol it is not, feel free to delete it. this is a leftover experimental way to add material via data attributes